sympa.spec

```diff
@@ -14,13 +14,12 @@
 %global unbundle_jquery_migrate 0
 %global unbundle_jquery_ui      0
 %global unbundle_jqplot         0
-%global unbundle_modernizr      0
 %global unbundle_respond        0%{?fedora}%{?rhel}
 
-#global pre_rel b.3
+%global pre_rel b.2
 
 Name:        sympa
-Version:     6.2.32
+Version:     6.2.33
 Release:     %{?pre_rel:0.}2%{?pre_rel:.%pre_rel}%{?dist}
```

Comment:
Nitpick, the '2' in the release field above should be '1' and the corresponding change should be applied to the changelog entry.
The release field needs to be reset to 1 (or 0.1 for a pre-release, but here the macros in the release field take care of the leading '0.' automagically) each time the version field is incremented (and also when the software goes from a pre-release to the final release, but this is not the case with Sympa, as next stable will be 6.2.34).